### ISSUE TITLE
Are you planning to publish OpenCV 3.3 to hub?

### ISSUE THREAD

**Issue by Kostanos** [julianbei/alpine-opencv-microimage#10]
I also recommend to leave OpenCV 2.4, as there are some project may require it. Like nodejs opencv library, they doesn't support OpenCV 3.x yet.

And thank you for this image!

**Comment by julianbei** [comment]
Yes it is planned but due to a small amount of spare time I cant promise when it is ready

**Comment by nalt** [comment]
+1
